tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/division_signpost_block.html

```diff
@@ -1,20 +1,27 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
-    {# Section heading #}
-    <h2>{{ value.title }}</h2>
-
-    {# Section intro #}
-    <div>{{ value.intro|richtext }}</div>
-
-    {# Signposting cards #}
-    <ul>
-        {% for card in value.cards %}
-            <li class="{{ card.card_colour }}">
-                {% firstof card.heading card.page.title %}
-                {{ card.description|richtext }}
-                {% image card.image fill-100x100 %}
-                <a href="{% pageurl card.page %}">{{ card.link_text }}</a>
-            </li>
-        {% endfor %}
-    </ul>
+<div class="grid division-signpost">
+    <div class="grid__division-signpost">
+        {# Section heading #}
+        {% if value.title %}
+            <h2 class="heading heading--two">{{ value.title }}</h2>
+        {% endif %}
+        {# Section intro #}
+        {% if value.intro %}
+            <p class="text text--five division-signpost__intro">{{ value.intro|richtext }}</p>
+        {% endif %}
+        {# Signposting cards #}
+        <ul class="division-signpost__cards">
+            {% for card in value.cards %}
+                <li class="division-signpost__card division-signpost__card--{{ card.card_colour }}">
+                    <div class="division-signpost__text">
+                        <h3 class="heading heading--two division-signpost__heading">{% firstof card.heading card.page.title %}</h3>
+                        <p class="heading heading--four-b heading--light division-signpost__description">{{ card.description|richtext }}</p>
+                    </div>
+                    {% srcset_image card.image format-webp fill-{540x280,490x280} sizes="(max-width: 598px) 540px, (min-width: 599px) 490px" class="" alt="" %}
```

Comment:
Question: Do we need the empty `class=""`?

Reply:
we don't:)